### PR TITLE
removing provider.tf

### PR DESCRIPTION
**Pull request by rfum**
# Contributing to the AWS Control Tower Account Factory for Terraform

Thank you for your interest in contributing to the AWS Control Tower Account Factory for Terraform.

At this time, we are not accepting contributions. If contributions are accepted in the future, the AWS Control Tower Account Factory for Terraform is released under the [Apache license](http://aws.amazon.com/apache2.0/) and any code submitted will be released under that license.

If you have a feature request, please create an issue using the Feature Request template, thanks!

### CODE REVIEW

**Comment by monteiro-renato-cog** [comment]
I still think we need to take a step back and look into this a bit better.
But if you had the green light from Robert, I will rubber stamp it to unblock you.
https://docs.aws.amazon.com/controltower/latest/userguide/account-troubleshooting-guide.html#w33aac36c31c45b7
**_Account Factory is an open source project. AWS Control Tower supports the Account Factory core code. If you make a local change to the Account Factory core code, AWS Control Tower only supports your Account Factory deployment on a best-effort basis._**

**Review by monteiro-renato-cog — APPROVED** [review]

**Comment by rfum** [comment]
@monteiro-renato-cog I've created an issue to the original repo with our requirements https://github.com/aws-ia/terraform-aws-control_tower_account_factory/issues/419 . I think we can wait for a while to their response.

**Pull request merged**
